README.md

```diff
@@ -1,96 +1,155 @@
 
-For a complete list of APIs and examples, please take a look at the [Python Client API Reference](https://min.io/docs/minio/linux/developers/python/API.html)
+This Quickstart Guide covers how to install the MinIO client SDK, connect to the object storage service, and create a sample file uploader.
 
-## Minimum Requirements
-Python 3.7 or higher.
+The example below uses:
+- [Python version 3.7+](https://www.python.org/downloads/) 
+- The [MinIO `mc` command line tool](https://min.io/docs/minio/linux/reference/minio-mc.html)
+- The MinIO `play` test server
 
-## Download using pip
+The `play` server is a public MinIO cluster located at [https://play.min.io](https://play.min.io).
+This cluster runs the latest stable version of MinIO and may be used for testing and development.
+The access credentials in the example are open to the public and all data uploaded to `play` should be considered public and world-readable.
+
+For a complete list of APIs and examples, see the [Python Client API Reference](https://min.io/docs/minio/linux/developers/python/API.html)
+
+## Install the MinIO Python SDK
+
+The Python SDK requires Python version 3.7+.
+You can install the SDK with `pip` or from the [`minio/minio-py` GitHub repository](https://github.com/minio/minio-py):
+
+### Using `pip`
 
 ```sh
 pip3 install minio
 ```
 
-## Download source
+### Using Source From GitHub
 
 ```sh
 git clone https://github.com/minio/minio-py
 cd minio-py
 python setup.py install
 ```
 
-## Quick Start Example - File Uploader
-This example program connects to an S3-compatible object storage server, make a bucket on that server, and upload a file to the bucket.
+## Create a MinIO Client
 
-You need the following items to connect to an S3-compatible object storage server:
+To connect to the target service, create a MinIO client using the `Minio()` method with the following required parameters:
 
-| Parameters | Description                                                |
-|------------|------------------------------------------------------------|
-| Endpoint   | URL to S3 service.                                         |
-| Access Key | Access key (aka user ID) of an account in the S3 service.  |
-| Secret Key | Secret key (aka password) of an account in the S3 service. |
+| Parameter    | Description                                            |
+|--------------|--------------------------------------------------------|
+| `endpoint`   | URL of the target service.                             |
+| `access_key` | Access key (user ID) of a user account in the service. |
+| `secret_key` | Secret key (password) for the user account.            |
 
-This example uses MinIO server playground [https://play.min.io](https://play.min.io). Feel free to use this service for test and development.
+For example:
 
-### file_uploader.py
 ```py
 from minio import Minio
-from minio.error import S3Error
 
+client = Minio("play.min.io",
```

Comment:
Perhaps this is more practical, less verbose yet still some additional context for novice readers.